File: sltn-examples/javascript/quick-start.js
// Sultan Chain - Zero Gas Quick Start

const { SultanSDK } = require('@sltn/sdk');

async function main() {
  // Connect - no API key needed!
  const sltn = new SultanSDK({
    network: 'mainnet',
    rpc: 'https://rpc.sltn.io'
  });

  // Check balance
  const balance = await sltn.getBalance('sultan1...');
  console.log(`Balance: ${balance} SLTN`);

  // Send transaction - ZERO gas fees!
  const tx = await sltn.sendTransaction({
    to: 'sultan1xyz...',
    amount: 100
    // Notice: no gas parameter needed!
  });
  console.log(`Transaction sent: ${tx.hash}`);
  console.log(`Gas fee: $0.00`);

  // Deploy a contract - also free!
  const contract = await sltn.deployContract({
    bytecode: '0x...',
    // No gas fees for deployment
  });
  console.log(`Contract deployed at: ${contract.address}`);
}

main().catch(console.error);

File: _archive/legacy-dirs/sltn-examples/python/quick_start.py
# Sultan Chain - Zero Gas Quick Start

from sltn import SultanSDK

def main():
    # Connect - no API key needed!
    sltn = SultanSDK('https://rpc.sltn.io')
    
    # Check balance
    balance = sltn.get_balance('sultan1...')
    print(f'Balance: {balance} SLTN')
    
    # Send transaction - ZERO gas fees!
    tx = sltn.send_transaction(
        to='sultan1xyz...',
        amount=100
        # No gas parameter needed!
    )
    print(f'Transaction sent: {tx.hash}')
    print(f'Gas fee: $0.00')

if __name__ == '__main__':
    main()
